tests/encrypted_lv.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { test, expect } from '@playwright/test';
```

Comment:
No, I missed the m because the way we are using the "playwright test" command with just words and not full name of the test files, the agama_dolomite_lvm test, was runnight both lvm.spec.ts and encrypted_lvm.spec.ts. I have added a comment here: https://progress.opensuse.org/issues/132989#change-663161  
Check playwright documentation: https://playwright.dev/docs/test-cli

Reply:
changed it and will put a new comment in Tina's ticket